packages/ai-visualize.js

```diff
@@ -6,6 +6,6 @@
  * https://opensource.org/licenses/MIT.
  */
 
-import { MCTSVisualizer } from '../src/ai/mcts-visualizer.js';
```

Comment:
I would suggest that we split this PR into two:

1. One that removes the `.js` suffices.
2. One that adds the typings.